### PR TITLE
Fixed #4797 : Peer review: Add one more choice to the list, and do not submit if no reason selected 

### LINKED ISSUE #4797: Peer review: Add one more choice to the list
peer review feature:

No FoP in copyright-related reasons for deletion (the second question). This is one of the most common reason for deletion, it should be in the list. Proposed message: "There is no freedom of panorama in the country where this picture was taken, so some installations (even permanently displayed in public spaces) can not be freely photographed and shared."

### PR DESCRIPTION
**Pull request by Rishavgupta12345**
Fixes #4797 
Peer review: Add one more choice to the list, and do not submit if no reason selected

### CODE REVIEW

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -510,6 +510,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="delete_helper_ask_reason_copyright_press_photo">Press photo</string>
   <string name="delete_helper_ask_reason_copyright_internet_photo">Random photo from internet</string>
   <string name="delete_helper_ask_reason_copyright_logo">Logo</string>
+  <string name="delete_helper_ask_reason_copyright_breach_of_freedom_of_panorama_rules">Breach of Freedom of Panorama rules</string>
```
[the remark this answers is not in the document]
Comment (Rishavgupta12345) [reply app/src/main/res/values/strings.xml]: done 😎 thanks

**Review by nicolas-raoul — COMMENTED** [review]
Thanks a lot! :-)

For UI pull requests, could you please send screenshots showing the results?
Here is what I see:
adb

**Comment by Rishavgupta12345** [comment]
> Thanks a lot! :-)
> 
> For UI pull requests, could you please send screenshots showing the results? Here is what I see: adb

I updated the copyright option and added reason 4 as No freedom of panorama which is working fine , but the scope option is also adding reason 4 as null , therefore this problem is happening ....  I will solve it soon

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul  I updated the copyright option and added reason 4 as No freedom of panorama which is working fine , but the (out of scope) option is also adding reason 4 as null , therefore this problem is happening .... 

can you please guide me how to solve this , I have tried but couldn't figure out .
for me the best option will be if I also add a Reason 4 for(out of scope), as you wish.

**Comment by nicolas-raoul** [comment]
Do you mean that in the current master, each of the step must have the exact same number of possible reasons?

That does not sound ideal, even though I guess that saves some lines of code.

How about designing a system that can take different numbers of reasons at each step?

**Comment by nicolas-raoul** [comment]
Could you please rebase on master? A related area has been changed in master. Thanks!

**Comment by Rishavgupta12345** [comment]
> Could you please rebase on master? A related area has been changed in master. Thanks!

yes done ,

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul  for now i am unable to remove the REASON 4 in out of scope option . I have tried many ways but it is not happenign . what should I do now ?

**Comment by nicolas-raoul** [comment]
How about designing a system that can take different numbers of reasons at each step?

Or feel free to unassign this issue and work on a different one if you prefer.

**Pull request closed without merging**